misc/ls-extensions/modules/project-design-service/src/main/java/io/ballerina/projectdesign/model/ModelElement.java

```diff
@@ -0,0 +1,24 @@
+package io.ballerina.projectdesign.model;
+
+import io.ballerina.tools.text.LineRange;
+
+import java.util.Optional;
+
+/**
+ * Represents the abstract model for a component model item.
```

Comment:
Let's add the `since` annotaton with the version

Reply:
Added